pkg/db-repo/change_history_manager.go

```diff
@@ -13,15 +13,16 @@ import (
 	"github.com/justtrackio/gosoline/pkg/funk"
 	"github.com/justtrackio/gosoline/pkg/log"
 	"github.com/justtrackio/gosoline/pkg/mdl"
+	"golang.org/x/exp/slices"
```

Comment:
I think the slices package is out of experimental and you can just import "slices" (might have a slightly different API though)